pulp_ansible/tests/functional/api/collection/v3/test_collection.py

```diff
@@ -221,6 +221,10 @@ def test_collection_version(artifact, pulp_client, collection_detail):
     assert "updated_at" in version
     assert "created_at" in version
 
+    assert "files" in version
```

Comment:
Is there some assertion that can be made for the data being returned? If we have a test that sync's a specific version of a collection then we could assert on the data populated during import.

Actually we could also do one to upload a collection and then assert there is on data currently and then when galaxy-importer releases it'll fail and we can update the test to match then.

Wdyt?

Reply:
actually `requires_ansible` is optional, even with galaxy-importer release it probably won't fail
(requires_ansible is rarely used)

Reply:
the importer is already released: https://github.com/ansible/galaxy-importer/blob/master/CHANGES.rst#galaxy-importer-0214-2021-01-28

Reply:
```
pulp_ansible/tests/functional/api/collection/v3/test_collection.py ...
>>>>>>>>>>>>>>>>>>>>>>>>>>> PDB set_trace (IO-capturing turned off) >>>>>>>>>>>>>>>>>>>>>>>>>>>
> /home/vagrant/devel/pulp_ansible/pulp_ansible/tests/functional/api/collection/v3/test_collection.py(224)test_collection_version()
-> assert "files" in version
(Pdb) version["files"]
{}
(Pdb) version["manifest"]
{}
(Pdb) version["requires_ansible"]
(Pdb) version
{'version': '1.0.0', 'href': '/pulp_ansible/galaxy/automation-hub/api/v3/collections/orionuser1/collection_dep_a_arwtsrzb/versions/1.0.0/', 'created_at': '2021-02-02T16:15:16.450378Z', 'updated_at': '2021-02-02T16:15:16.450397Z', 'manifest': {}, 'files': {}, 'requires_ansible': None, 'artifact': {'filename': 'orionuser1-collection_dep_a_arwtsrzb-1.0.0.tar.gz', 'sha256': '35ce4f9e17a9d973939e044b7eedd4355f93ee827dd36104bbb6890afed44eb7', 'size': 1520}, 'collection': {'id': 'cda47603-fcbb-4b84-87e2-d9631979a33d', 'name': 'collection_dep_a_arwtsrzb', 'href': '/pulp_ansible/galaxy/automation-hub/api/v3/collections/orionuser1/collection_dep_a_arwtsrzb/'}, 'download_url': 'http://pulp3-source-fedora33.localhost.example.com/pulp/content/automation-hub/orionuser1-collection_dep_a_arwtsrzb-1.0.0.tar.gz', 'name': 'collection_dep_a_arwtsrzb', 'namespace': {'name': 'orionuser1'}, 'metadata': {'authors': ['Orion User 1'], 'contents': [], 'dependencies': {}, 'description': 'a collection with some deps on other collections', 'documentation': '', 'homepage': '', 'issues': '', 'license': ['GPL-3.0-or-later'], 'repository': 'http://github.example.com/orionuser1/skeleton', 'tags': ['collectiontest']}}
```

Reply:
Yes so what I'm saying is that in this response we're asserting on the key and not the value. The test data has `'files': {}` so asserting on `{}` isn't worth much. What would be better is to have the test data populate the `files` `manifest` and `requires_ansible` responses with actual data and then the test code assert on the values (not just the presence of the keys).

Reply:
Oh I see a test further down that seems like it's doing just that. Let me check that out.